samples/hello-world-entity/src/test/java/org/terracotta/management/entity/helloworld/HelloWorldTest.java

```diff
@@ -1,19 +1,19 @@
 /*
  * Copyright Terracotta, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I just meant that because of IDE formatting, you introduced a 10 lines change instead of 1 line change :-)